Makefile

```diff
@@ -1,16 +1,22 @@
-.PHONY: all image package dist clean
+.PHONY: all image package dist deploy clean
 
-all: package
+all: image dist
 
 image:
 	docker build --tag amazonlinux:nodejs .
 
-package: image
-	docker run --rm --volume ${PWD}/lambda:/build amazonlinux:nodejs npm install --production
+awsconfigure:
+	docker run -it --rm --volume ${PWD}/../.aws:/root/.aws amazonlinux:nodejs aws configure
+
+package:
+	docker run --rm --volume ${PWD}/lambda:/build --volume imgresize-modules:/build/node_modules amazonlinux:nodejs npm install --production
```

Comment:
So setting a separate volume mount for the node modules directory seems to be the crux of this pull. I need to spin up a Windows box with Docker to test and better understand why this is necessary.